docs/General/Manuals/client-manual.rst

```diff
@@ -0,0 +1,1638 @@
+button, a confirmation dialog will appear. In Figure 47's example a :improvement:`Granary` is being
+constructed for a total cost of 30 shields (production). The city has produced net 17 of 30 needed shields and
+at its current rate of net +4 production will take 4 turns to complete. The player for this example has also
+added multiple items to the work list. When the :improvement:`Granary` is finished, the city will start
+production on :improvement:`Barracks`. At this point, the city only needs 13 shields to finish the
+:improvement:`Granary`, but at +4 shields per turn will produce a total of 16 shields in the same 4 turns.
+This means that the surplus shields will transfer to the :improvement:`Barracks` when the
+:improvement:`Granary` is complete.
+
+.. figure:: ../../_static/images/gui-elements/city-dialog-prod.png
+  :align: center
+  :alt: Freeciv21 City Dialog Production
+  :figclass: align-center
+
+  Figure 47: City Dialog - Production
+
+
+At the bottom of the segment are four buttons: :guilabel:`Add`, :guilabel:`Move Up`, :guilabel:`Move Down`,
+and :guilabel:`Delete`. Clicking on :guilabel:`Add`, will open a pop up menu showing all of the items that the
+city can produce. Clicking on the item will add it to the bottom of the work list above. If you wish to move
+this new item or another item in the work list up, click (select) it from the work list and click the
+:guilabel:`Move Up` button. Same action for the :guilabel:`Move Down` button. To remove the item from the work
+list, select an item in the work list and click on :guilabel:`Delete`. You can also double-click on an item in
+the work list and it will be removed from the work list.
+
+One more feature of the :guilabel:`Production` tab is the work list editor. You can save work lists for later
+use. To get started, populate the work list with things you want to build. This can be a collection of city
+improvements, units and wonders. Once you have the list configured the way you like it, right-click on the
+work list canvas and a pop up menu will show. Start by selecting :guilabel:`Save Worklist`. A dialog box will
+pop up allowing you to give the list a name. If you clear out the work list on the :guilabel:`Production` tab
+and then right-click on the work list canvas, you can pick :guilabel:`Insert Worklist` or :guilabel:`Change
+Worklist` from the sub-menu. :guilabel:`Insert Worklist` will add the items in the saved work list to the main
+work list. :guilabel:`Change Worklist` will clear what is in the main work list and replace it with the saved
+work list.
+
+Moving to the center, we can see the city citizen tile output segment and the city's full vision radius
+highlighted over the main map as highlighted in Figure 48. This city is size 3, which means 4 tiles can be
+worked by the citizens. The city center is worked for free and then 3 other tiles are being worked. In this
```

Comment:
(Later) I find this a bit confusing, is it 3 or 4 tiles in the end?

Reply:
changed to show three tiles, stating the city tile is always worked for free